iree/compiler/Conversion/LinalgToSPIRV/ConvertToGPUPass.cpp

```diff
@@ -431,6 +430,24 @@ static void getGPUProcessorIdsAndCounts(Location loc,
   }
 }
 
+template <typename GPUIdOp, typename GPUCountOp>
+static ProcessorIdAndCount getLinearizedGPUProcessorIdAndCount(
+    Location loc, ConversionPatternRewriter &rewriter) {
+  std::array<Value, 3> ids, counts;
+  getGPUProcessorIdsAndCounts<GPUIdOp, GPUCountOp>(loc, rewriter, 3, ids,
```

Comment:
would be great if we declare a constant var for `3`, maybe just `kNumDims`?

And we can also reuse it in the later for-loop, to avoid magic number.